lang/en/docs/tutorials/python-ml/predict-ml-properties.md

```diff
@@ -93,15 +95,15 @@ We can now [run the job](../../jobs/actions/run.md) and wait for it to complete.
 
 ## 7. Analyze the Prediction Results
 
-After a few minutes, the job will complete. In the [files tab](../../jobs/ui/files-tab.md), we can find a new file
-called "predictions.csv", which contains the predictions generated by this model. Under the hood, this model is created
-inside of the Train/Predict workflow unit.
+After a few minutes, the job will complete. We can then visit the job's [results tab](../../jobs/ui/results-tab.md),
+where we will see a CSV preview of a file called `predictions.csv`. These are the row-by-row predictions generated by
+the model. Under the hood, this file is generated inside the `Model Train and Predict` unit.
 
 ## Animation
 
 This tutorial is demonstrated in the following animation:
 
 <div class="video-wrapper">
-<iframe class="gifffer" width="100%" height="100%" src="https://www.youtube.com/embed/fRjFUSy50RY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<iframe class="gifffer" width="100%" height="100%" src="https://www.youtube.com/embed/NWlv5Fsum-0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Let's change the slide that is used for the video thumbnail pls to be more descriptive

Reply:
Re Video:

- "First, click the Dropbox **link** in the left-hand sidebar", not "button"

Reply:
Changed the phrasing in the video, new version doesn't have the JSON editor